bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/productlist/ProductListImpl.java

```diff
@@ -15,10 +15,10 @@
 package com.adobe.cq.commerce.core.components.internal.models.v1.productlist;
```

Comment:
The changes made to this class are almost entirely in an effort to port the functionality from the `SearchResults` component here, so that essentially the same functionality available in search is available on the product listing pages (basically the category pages)